PROPOSALS.md

```diff
@@ -115,6 +117,20 @@ request feedback on feasibility before investing time and effort in the full pro
     * all changes to `base` package slated for future release should land in GHC `master` before the next major release branch is forked (also see [GHC wiki on major releases](https://gitlab.haskell.org/ghc/ghc/-/wikis/GHC-status#1-major-releases))
       - backports are usually reserved for security or packaging matters
 
+If your proposal is a "multi-phase proposal" with multiple implementation and deprecation steps, make sure to
+precisely define when each step is supposed to take place.
```

Comment:
Presumably the "when" is likely to be e.g. "in base-4.28" (or "in the next major base version after the previous step" would be more precise but I don't think we need them to be that pedantic). Might be worth clarifying that so that it doesn't look like we're asking for a date? Not sure if that confusion is likely.

Reply:
I don't want to go into too much detail here.

In general: when something is to be released is up to GHC HQ, not the proposer. The proposer is however responsible for getting something **merged**. That indeed complicates deprecation phases, but that's something we will have to play by ear.